### DISCUSSION TITLE
正则/123.+/ 可以匹配 "123\nabc"吗？

### DISCUSSION THREAD

**Discussion by ruozhiertong** [Bowen7/regex-vis#30] [Q&A]
正则/123.+/ 可以匹配 "123\nabc"吗？
按照一般正则的教程中，. 是没有匹配换行\n的。 
在作者项目中，发现这个字符串可以匹配到。 作者是默认使用了s标志 即/123.+/s  吗?

**Comment by Bowen7** [comment]
你是在 Test 下面进行测试的吗

Test 下的 Input 并不支持多行输入（可能换成 Textarea 更合理？）。你输入的 `123\nabc` 实际上是 `123\\nabc`

**Comment by ruozhiertong** [comment]
是的。
我在其他平台上的测试，没有问题，它们用的应该是textarea。
![屏幕快照 2022-04-14 00 51 18](https://user-images.githubusercontent.com/24388007/163231918-c0e7f0ca-47eb-4c29-9241-72efb505f73f.png)
![屏幕快照 2022-04-14 00 52 54](https://user-images.githubusercontent.com/24388007/163231957-639d1440-6ab6-4510-9650-616105c89186.png)
![屏幕快照 2022-04-14 00 52 41](https://user-images.githubusercontent.com/24388007/163231974-95798178-5b7d-48a1-8042-fbee672e67aa.png)
![屏幕快照 2022-04-14 00 53 10](https://user-images.githubusercontent.com/24388007/163232004-d529e88f-cf0f-43c3-bdbc-e16534e018b3.png)

**Reply by Bowen7** [reply]
我将 Test 下的 Input 改成 Textarea 了，现在支持输入多行文本了
